### DISCUSSION TITLE
onSlideIsOpenChanged in v1.0

### DISCUSSION THREAD

**Discussion by d6p2b** [letsar/flutter_slidable#263] [General]
Thanks very much for the update to v1.0 along with the migration guide.   I'm working through my migration and am not quite sure how to move forward with code which used to react to the v0.6.0 onSlideIsOpenChanged calls in the SlideableController.  Can you offer any suggestions or maybe update the migration guide?  Thanks!!!

**Comment by letsar** [comment]
Hi @d6p2b! In order to make it more flexible, I redesign that part. For the moment, there is no built-in way to replace `onSlideIsOpenChanged`.
- If the widget is a child of `Slidable`, you can create a widget which get the current `SlidableController` (with `Slidable.of(context)`) and listen to the `actionPaneType` notifier and get which pane is currently open.
- If the widget is outside the Slidable widget tree, then you'll need to have a child of a Slidable that listen to the `actionPaneType` notifier (as in the previous bullet point), but instead of acting on this widget, you'll need to pass the information to an external widget.

I can provide you a more detailed solution, if you can describe me what you want to achieve.

**Reply by PetrachkovArtem** [reply]
Hi, how i can get event about that a slide is open?
